src/main/java/soolutions/hometaskapp/Account.java

```diff
@@ -0,0 +1,7 @@
+package soolutions.hometaskapp;
+
+public interface Account {
+    boolean open();
+
+    int balance();
```

Comment:
This interface seems designed in such a way that it will likely grow and change. Do we want to accumulate more functions on the Account interface?